src/Assets/Api.php

```diff
@@ -40,7 +40,7 @@ public function __construct( Package $package ) {
 	 * @return string The cache buster value to use for the given file.
 	 */
 	protected function get_file_version( $file ) {
-		if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) {
+		if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG && file_exists( $this->package->get_path() . $file ) ) {
```

Comment:
good additional change here 👍 .  Although I wonder if we should still log instances when the file doesn't exist? Should there be at least a log trail of filemtime failing because the file doesn't exist for potential troubleshooting?

Reply:
I'm not sure to understand you. You mean throwing an error if the file doesn't exist?

Reply:
No I mean `error_log` if it doesn't exist. Previously there'd be a php warning if `filemtime` was used on a file that doesn't exist and that'd be a signal that something went wrong with the build. So my comment was birthed out of wondering if we're okay with losing that signal that something isn't right with the build process.

I'm on the fence here, and I'd be okay with leaving this code as is.